indico/web/forms/fields.py

```diff
@@ -249,6 +249,21 @@ def __init__(self, *args, **kwargs):
         super(IndicoPasswordField, self).__init__(*args, **kwargs)
 
 
+class CategoryField(HiddenField):
+    """Field that lets you select a category"""
+
+    widget = JinjaWidget('forms/category_picker_widget.html')
+
+    def process_formdata(self, valuelist):
+        from indico.modules.categories import Category
+        if valuelist:
+            category_id = json.loads(valuelist[0])
```

Comment:
I still don't see why you are using `json.loads()` here. If you always expect an int just use `int(valuelist[0])` :)